.changeset/stupid-maps-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ag.ds-next/header': minor
+---
+
+Add HeaderBadge
```

Comment:
```suggestion
Add badge
```

Reply:
We don't need to say `HeaderBadge` here as the changeset relates this to the `header` package